desktop/lib/window-manager/index.js

```diff
@@ -63,26 +64,35 @@ function setDimensions( config ) {
 	return config;
 }
 
-function openWindow( windowName ) {
+async function openWindow( windowName ) {
 	if ( windows[windowName] ) {
 		let settings = windows[windowName];
 
 		if ( settings.handle === null ) {
-			Config[settings.config] = setDimensions( Config[settings.config] );
+			let config = Config[settings.config];
+			config = setDimensions( config );
 
-			windows[windowName].handle = new BrowserWindow( Config[settings.config] );
-			windows[windowName].handle.setMenuBarVisibility( false );
-			windows[windowName].handle.webContents.session.setProxy( { proxyRules: 'direct://' }, function() {
-				windows[windowName].handle.loadURL( Config.server_url + ':' + Config.server_port + '/desktop/' + settings.file );
+			// nodeIntegration is necessary to support usage of `require` in preload and HTML scripts.
+			// FIXME: Refactor preload to use contextBridge instead (contextIsolation: true, nodeIntegration: false).
```

Comment:
An earlier draft of this upgrade went down the route of using `contextBridge`. However I wanted to keep the diff of this PR somewhat reasonable, and that work will be part of a downstream PR.

Reply:
It is for sure safer to have it off but the assets are bundled so there should not be an issue here.